### ISSUE TITLE
Rounded Corners for the Always on Top Window

### ISSUE THREAD

**Issue by tinyplayerss** [3dproger/AxelChat#351]
I love this Tool its a Cool Chat for my Streams and That i love this Tool I feel like there is Some Preference Settings missing like Adding Rounded Corners or Allow us to make Changes to the Window Via a Config or Something in the Settings, Thank You for making this Tool and Please Add some Rounded Corner Settings!

**Comment by 3dproger** [comment]
Hello! Thanks for your feedback! But unfortunately, I don’t quite understand what rounded corners we are talking about. Do you mean the case when the window frame is turned off, as in the screenshot below?
![изображение](https://github.com/3dproger/AxelChat/assets/22375229/0a95e3cc-5b71-4bef-8ade-1c89310b4c7c)

**Comment by tinyplayerss** [comment]
> Hello! Thanks for your feedback! But unfortunately, I don’t quite understand what rounded corners we are talking about. Do you mean the case when the window frame is turned off, as in the screenshot below? ![изображение](https://private-user-images.githubusercontent.com/22375229/303929909-0a95e3cc-5b71-4bef-8ade-1c89310b4c7c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc2NzQ2NTMsIm5iZiI6MTcwNzY3NDM1MywicGF0aCI6Ii8yMjM3NTIyOS8zMDM5Mjk5MDktMGE5NWUzY2MtNWI3MS00YmVmLThhZGUtMWM4OTMxMGI0YzdjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjExVDE3NTkxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWExY2RkN2YzMTA4OGY0Njc5OGMyZTE4MTFjNDg2NDQwMWJhY2QzMmVlYTI0NGYyNTNhMDEzNWFlM2IwY2Q0YjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RV9ZTwjqeEQs-NRnOfbhJgH0iPC7RXh_tZhq1CXL0ho)

Yes when the Window Frame is Off then it Should be Rounded Corners but Give us the Ability to change the Radius size of the Rounded Corners.

**Comment by 3dproger** [comment]
Okay, I'll try to do that in the future then 😉
